### PR TITLE
two folders translated, rest will follow later

### PR DESCRIPTION
**Pull request by FredMark**
I translated all the files in 2 of the folders.
The other files/folders will follow later.

### CODE REVIEW

**Comment by xlcrr** [comment]
Nice work! We will merge this into master when its ready

**Comment by FredMark** [comment]
This PR still seems to be blocked by 1 approving review.
Can anybody do that?

**Review by xlcrr — APPROVED** [review]

**Pull request merged**
